openshift/templates/web/config/nginx.conf.template

```diff
@@ -74,11 +78,13 @@ http {
         default $http_x_forwarded_server;
         '' $host;
     }
+    # ======================================================
 
     server {
-        listen       8080;
-        server_name  localhost;
-
+        %LISTEN_CONFIG_SECTION%
```

Comment:
I give, how does this work?

Reply:
Works in conjunction with https://github.com/bcgov/sheriff-scheduling/blob/master/docker/nginx-runtime/s2i/bin/run.
This is a copy of https://github.com/bcgov/sheriff-scheduling/blob/master/docker/nginx-runtime/nginx.conf.template that is mounted as a config map so it can be edited in OCP for troubleshooting purposes.